src/SyncStorage.re

```diff
@@ -9,7 +9,7 @@ let refreshToken = maybeToken =>
   | Some(jwt) =>
     let token = Js.Dict.empty();
     Js.Dict.set(token, "token", Js.Json.string(jwt));
-    Sync.set(token, () => ());
+    Chrome.Extensions.Storage.Sync.set(token, () => ());
```

Comment:
Why calling the fn by its full name if there is already an `open Chrome.Extensions.Storage`

Reply:
oh fuck I was just trying things and forgot to remove it